src/ui/units/workbooks/store/actions/index.ts

```diff
@@ -417,23 +417,46 @@ type AddWorkbookInfoAction = {
         workbookId: string;
         workbookName: string;
         workbookPermissions: WorkbookPermission;
+        workbookBreadcrumbs?: GetCollectionBreadcrumbsResponse | null;
     };
 };
 
-export const addWorkbookInfo = (workbookId: string) => {
-    return (dispatch: WorkbooksDispatch) => {
-        getSdk()
-            .us.getWorkbook({workbookId, includePermissionsInfo: true})
-            .then((workbook) => {
+export const addWorkbookInfo = (workbookId: string, withBreadcrumbs = false) => {
+    return async (dispatch: WorkbooksDispatch) => {
+        const workbook = await getSdk().us.getWorkbook({workbookId, includePermissionsInfo: true});
+
+        let workbookBreadcrumbs = null;
+
+        if (withBreadcrumbs && workbook.collectionId) {
```

Comment:
If workbook isn't in a collection, store will remain empty (and the same will happen if we don't pass the `withBreadcrumbs` argument):

<img width="399" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/bd5bc633-78db-46ee-9cc2-b43b7372b933">

Reply:
yea, fixed these with flag requestedWithBreadcrumbs